tools/GenerateExternalContributors.ps1

```diff
@@ -0,0 +1,123 @@
+<#
+.Synopsis
+Get all extenal contibuting authors.
+.Description
+Get all extenal contibuting authors.
+.Outputs
+The name, login, commits message of the authors.
+.Link
+Invoke-WebRequest: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-7
+Invoke-RestMethod: https://docs.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Utility/Invoke-RestMethod?view=powershell-7
+#>
+param(
+    [Parameter(Mandatory)]
+    [string]
+    $AccessToken
+)
+$SinceDate = (Get-Date).AddDays(-28)
```

Comment:
Could you add an optional variable? The default value is 28.

Reply:
Done